brave/brave-core
# Android safebrowsing safetynet

Resolves https://github.com/brave/brave-browser/issues/8705

Security review: https://github.com/brave/security/issues/1118

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
1. Make sure that `brave://flags/#brave-safe-browsing` is Enabled
2. Make sure that `Safe Browsing` is enabled in `Settings->Brave Shields & privacy`
3. Visit https://testsafebrowsing.appspot.com/ and open diff links comparing results with Chrome on Android (only the _Webpage warnings_ and _IOS/OSX Warnings_ sections).
4. Make sure Brave has a parity with Chrome.
![Screenshot_1669305405](https://user-images.githubusercontent.com/12011303/203825974-698dc7a1-defe-4673-b97c-75872924c915.png)

![Screenshot_1669305435](https://user-images.githubusercontent.com/12011303/203825984-74bb0ed1-71d4-4903-9aed-a6ad23bfddd7.png)
